workload-team/cluster-stamp.bicep

```diff
@@ -1786,6 +1786,8 @@ resource mc 'Microsoft.ContainerService/managedClusters@2024-03-02-preview' = {
     enablePodSecurityPolicy: false
     networkProfile: {
       networkPlugin: 'azure'
+      networkPluginMode: 'overlay'
```

Comment:
Does this change, in any way, affect the details on <https://github.com/johndowns/aks-baseline/blob/cni-overlay/network-team/topology.md>?

Reply:
Technically, no - but I think it's a good idea to add some new content to that file to explain what's going on with this extra CIDR range. I've pushed an update.